src/test/regress/multi_schedule

```diff
@@ -41,7 +41,7 @@ test: multi_read_from_secondaries
 # ----------
 test: multi_create_table
 test: multi_create_table_superuser
-test: multi_create_table_constraints multi_master_protocol multi_load_data multi_load_data_superuser multi_behavioral_analytics_create_table
+test: multi_create_table_constraints multi_master_protocol multi_load_data multi_load_data_superuser multi_behavioral_analytics_create_table multi_create_table_triggers
```

Comment:
when 6 tests are running in parallel, it sometimes become slower with small machines. Can we move this test to somewhere else to run with less concurrency?